packages/code-du-travail-frontend/src/conventions/texte/Sidebar.js

```diff
@@ -2,20 +2,24 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Section, Wrapper } from "@cdt/ui";
 import SummaryTitle from "./SummaryTitle";
+import { TocList } from "./toc";
 
-const Sidebar = ({ node, onSummaryTitleToggleExpanded }) => {
+const Sidebar = ({ node, onSummaryTitleToggleExpanded, tocbotEnabled }) => {
   return (
-    <Section>
+    <Section className="js-toc">
```

Comment:
I guess `js-toc` is  the default value here when initializing Toc, 
Can you add a comment here to let "futur us" know that it tocbot related ?